google/generativeai/embedding.py

```diff
@@ -34,16 +34,24 @@
 _EMBED_CONTENT_DOC = """Calls the API to create embeddings for content passed in.
 
     Args:
-        model: Which model to call, as a string or a `types.Model`.
+        model:
+            Which [model](https://ai.google.dev/models/gemini#embedding) to
```

Comment:
This is the only update. The rest is just a drive-by fix of the formatting to make each line less than 80 char width.

The deeplink to the `#embedding` section might rot over time but as long as the https://ai.google.dev/models/gemini continues to be the correct page to link to then it should be fine. Worst case, the browser just shows you the top of the page and you have to scroll down to find the embeddings model info. Best case, the deep link continues to work and we jump users to exactly the info they need.

Reply:
Interesting discovery that this docstring technique we're using means we are side-stepping `black` checks for docstrings. Everything gets line-length checks, but herestrings would be mutilated if they were re-wrapped.

`Return` below should be `Returns` too :shrug:

Reply:
Good point, we only did this for the "async_code_match_test" which forces the implementations of the sync and async versions to be identical except for key words like async, await,...

We used a better solution for the semantic retreiver where we just stripped the """ strings before doing the comparison. 

I'll go fix that.

Reply:
@The most important aspect is the ability for you and your team members and

Reply:
ios